src/App.js

```diff
@@ -37,4 +48,4 @@ function App() {
   }
 }
 
-export default App;
+export default withFirestore(App);
```

Comment:
I forgot to delete `withFirestore`. Could you remove this and its import before merging?